deploy/cr.yaml

```diff
@@ -538,10 +538,10 @@ spec:
 #    resources:
 #      limits:
 #        cpu: "300m"
-#        memory: "0.5G"
+#        memory: "1.2G"
 #      requests:
 #        cpu: "300m"
-#        memory: "0.5G"
+#        memory: "1G"
```

Comment:
Maybe we should set these defaults in this method if the memory is less than 1G: https://github.com/percona/percona-server-mongodb-operator/blob/1091ea0fe4241d4668e544611a40189be01b9c57/pkg/apis/psmdb/v1/psmdb_defaults.go#L57?

Reply:
@pooknull @inelpandzic We need to overwrite the user's settings. If the cluster does not have enough memory, the pod will not be started. PBM can't work correctly if it does not have enough memory.